setup.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
```

Comment:
No need for this try/except block now since distutils doesn't support an equivalent find_packages.